runelite-client/src/main/java/net/runelite/client/config/ConfigManager.java

```diff
@@ -692,7 +692,7 @@ public void setDefaultConfiguration(Object proxy, boolean override)
 			return;
 		}
 
-		for (Method method : clazz.getDeclaredMethods())
+		for (Method method : clazz.getMethods())
```

Comment:
Actually, this is potentially a breaking change because `getMethods()` returns public methods only (including inherited ones), whereas `getDeclaredMethods()` returns public, protected, package, and private methods (excluding inherited ones).

I think I'll have to add a function to just do DFS on the interface and call `getDeclaredMethods()` on all of the superinterfaces to prevent a breaking change.